src/Controller/RentalController.java

```diff
@@ -0,0 +1,268 @@
+            }
+            int halfRent = artworkRentPrice / 2;
+            if (halfRent % 2 != 0) {
+                System.out.println("Rent can only be fully paid. No more half rents.");
+            }
+            System.out.println("Please select how much you want to pay from the following options: \n 1. Full Rent \n 2. Half Rent");
+            int userChoice = scanner.nextInt();
+            if (halfRent % 2 != 0) {
+                userChoice = 1;
+            }
+            switch (userChoice) {
+                case 1:
+                    done = false;
+                    while (!done) {
+                        System.out.println("Please enter the amount to pay");
+                        int userMoney = scanner.nextInt();
+                        if (userMoney > artworkRentPrice) {
+                            System.out.println("Your change is: " + (userMoney - artworkRentPrice));
+                            done = true;
+                        } else if (userMoney < artworkRentPrice) {
+                            System.out.println("Insufficient Payment. Please try again");
+                        }
+                    }
+                    System.out.println("Rent has been paid");
+                    instituteList.get(artworkLocation).setRentalStatus("paid");
+                    instituteList.get(artworkLocation).setPendingRental(0);
+                    done = true;
+                    break;
+                case 2:
+                    done = false;
+                    while (!done) {
+                        System.out.println("Please enter the amount to pay");
+                        int userMoney = scanner.nextInt();
+                        if (userMoney > halfRent) {
+                            System.out.println("Your change is: " + (userMoney - halfRent));
+                            done = true;
+                        } else if (userMoney < halfRent) {
+                            System.out.println("Insufficient Payment. Please try again");
+                        }
+                    }
+                    System.out.println("Rent has been paid");
+                    instituteList.get(artworkLocation).setRentalStatus("pending");
+                    int nwDebt = halfRent/2;
+                    instituteList.get(artworkLocation).setPendingRental(nwDebt);
+                    done = true;
+                    break;
+                default:
+                    System.out.println("That is not a valid option");
+            }
+        }
+    }
+
+    /**
+     * Method to print the defined rented prince of an Artwork
+     */
+    public void rentedPrice() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Please select the Artwork: ");
+        artworkControllerInstance.showArtworks();
+        String artworkName = scanner.next();
+        String artworkLocation = artworkControllerInstance.getArtworkList().get(artworkName.toLowerCase()).getInventoryLocation();
```

Comment:
do this in steps so you can properly validate that artwork is not null before trying the getinventoryloc